lib/galaxy/webapps/galaxy/api/tools.py

```diff
@@ -387,7 +387,7 @@ def _rerun_tool( self, trans, payload, **kwargs ):
         # job's previous parameters and incoming parameters. Incoming parameters
         # have priority.
         #
-        original_job = self.hda_manager.creating_job( trans, original_dataset )
+        original_job = self.hda_manager.creating_job( original_dataset )
```

Comment:
:+1: This was broken in fafea361ac05f8787d271ebab1b777294592b46b and needs to be backported to release_15.05 .